modules/phase_field/doc/content/source/auxkernels/ADTotalFreeEnergy.md

```diff
@@ -0,0 +1,24 @@
+# ADTotalFreeEnergy
+
+AD version of TotalFreeEnergy that uses AD material properties
```

Comment:
```suggestion
AD version of [TotalFreeEnergy.md] that uses AD material properties
```

Reply:
I had it like this originally, but it kept failing yesterday in one of the checks here in "magpie". It couldn't find the link somehow and failed. So I ended up removing it.

Reply:
sounds good. magpie must be missing that part of the documentation smh